### ISSUE TITLE
"Copy GitHub URL with selection" doesn't work

### ISSUE THREAD

**Issue by Gama11** [Cycloware/vscode-github-file-folder-url#2]
When running that command, no matter whether I have anything selected or not, I get the following error:

```
Running the contributed command:'extension.github-file-folder-url.copyGithubUrlWithSelection' failed.
```

**Comment by KavanShaban** [comment]
I'm starting to look at this now.

**Comment by KavanShaban** [comment]
This is now fixed in `1.1.8`

**Comment by Gama11** [comment]
Thanks for the quick fix! 👍
